src/index.js

```diff
@@ -22,11 +22,13 @@ const server = http.createServer((req, res) => {
         const notification = JSON.parse(body);
         const chatId = CHAT_IDS_MAPPING[req.url];
 
-        if(Object.keys(notification).includes('image')) {
+        const message = notification.text !== undefined ? notification.text : notification.message;
+
```

Comment:
but what if `message` is still `undefined`?
would you throw an exception>

Reply:
Fixed